libafl_intelpt/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "libafl_intelpt"
+version.workspace = true
+authors = ["Marco Cavenati <cavenatimarco@gmail.com>"]
+description = "Intel Processor Trace wrapper for libafl"
+repository = "https://github.com/AFLplusplus/LibAFL/"
+edition = "2021"
+license.workspace = true
+readme = "./README.md"
+keywords = ["fuzzing", "testing", "security", "intelpt"]
+categories = ["development-tools::testing", "no-std"]
+
+[features]
+default = ["std", "libipt"]
+std = ["libafl_bolts/std"]
+
+libipt = ["std", "dep:libipt"]
+
+[dev-dependencies]
+static_assertions = { workspace = true }
+
+[target.'cfg(target_os = "linux" )'.dev-dependencies]
+nix = { workspace = true }
+proc-maps = "0.4.0"
+
+[dependencies]
+arbitrary-int = { version = "1.2.7" }
+bitbybit = { version = "1.3.2" }
```

Comment:
This comment still stands, we should try to keep the amount of dependencies as low as possible